providers-sdk/v1/lr/headers.go

```diff
@@ -0,0 +1,31 @@
+package lr
+
+import (
+	"strings"
+	"text/template"
+)
+
+const defaultLicenseHeaderTpl = `{{.LineStarter}} Copyright (c) Mondoo, Inc.
```

Comment:
do we need the line starter to be a separate, specifiable param? why not just bake it in?

Reply:
We have yaml and we have go files that are generated, so // vs # for comments
Imo this is more extendable if we every need that in the future